web/src/components/ParticipantForm/ParticipantFormStates/PFS.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import axios from '../../../axios-orders';
+
+import { FormGroup, InputGroup } from "@blueprintjs/core";
+
+class PFS extends Component {
+    state = {
+        email: '',
+        name: '',
+    }
+
+    submitHandler = (event) => {
+        event.preventDefault();
+        const aForm = {
+            // myState = this.state.email,
+            // myName = this.state.name,
+            customer: {
+                email: 'erdete01@luther.edu',
+                name: 'Temuulen Erdenebulgan'
+            }
+        }
+        axios.post( '/referral.json', aForm)
+            .then(response => console.log(response))
+            .catch(error => console.log(error));
+    }
+
+    render() {
+        return (
+            <div>
+                <h1>Participant Form</h1>
+                <p>Welcome to Referral Program! Our mission is to help you find the best position</p>
+                <FormGroup
+                    label="Email"
+                    labelFor="text-input"
+                    labelInfo="(required)"
+                    inline={true} >
+                    <InputGroup id="text-input" placeholder="Placeholder text" />
+                </FormGroup>
+                <FormGroup
+                    label="Hello! It's nice to e-meet you. What's your name? (First name, Last name)"
+                    labelFor="text-input"
+                    labelInfo="(required)"
+                    inline={true}>
+                    <InputGroup id="text-input" placeholder="Placeholder text" />
+                </FormGroup>
+                <button clicked={this.submitHandler}>Submit</button>
```

Comment:
Can we make this button the same as the button on the Referrer form?
<img width="444" alt="Screen Shot 2020-07-21 at 10 34 55 AM" src="https://user-images.githubusercontent.com/1626382/88081731-0c1f2500-cb3e-11ea-8ca1-a6fb69c20a96.png">
<img width="414" alt="Screen Shot 2020-07-21 at 10 34 38 AM" src="https://user-images.githubusercontent.com/1626382/88081733-0d505200-cb3e-11ea-98fe-eab0a8ef99ca.png">

Reply:
Changed it